lib/spack/spack/spec.py

```diff
@@ -860,11 +885,21 @@ class SpecBuildInterface(ObjectWrapper):
         default_handler=_headers_default_handler
     )
 
+    all_headers = ForwardQueryToPackage(
+        'all_headers',
+        default_handler=_all_headers_default_handler
+    )
+
     libs = ForwardQueryToPackage(
         'libs',
         default_handler=_libs_default_handler
     )
 
+    all_libs = ForwardQueryToPackage(
```

Comment:
you don't need this for `gnutls`.

Reply:
Yeah, I saw that - that was my bad. I'm going to rebase the gnutls branch to start from develop instead of my other branch, [v-dobrev:props_all_libs_all_headers](https://github.com/v-dobrev/spack/tree/props_all_libs_all_headers).

Reply:
or just `git rest --soft develop` and then reset the files you don't need.

Reply:
Oops, I already pushed the rebased branch. Is that OK?

Reply:
sure